### DISCUSSION TITLE
Streak Bug, where it show's contribution for me but the streak show otherwise

### DISCUSSION THREAD

**Discussion by Arpitchugh** [community/community#31924] [Other Feature Feedback, Questions, & Ideas] [answered]
If another user check my profile it shows that no contribution is made on 1september, 2022, and the streak is broken on that day, but I made a contribution on this day and my profile also shows the same
### screenshot of my work on 1 September
![image](https://user-images.githubusercontent.com/63435960/188314908-4819484e-0b5b-4a20-a262-5f56383960da.png)
### screenshot when the streak is broken
![image](https://user-images.githubusercontent.com/63435960/188314915-eccf3393-3f34-440d-9013-85e33455e39a.png)
![image](https://user-images.githubusercontent.com/63435960/188314935-7e8c6244-93dd-4156-b903-016af168c394.png)

**Comment by DenverCoder1** [comment, marked as answer]
This is currently a limitation of the GitHub API.

The contribution graph is shown in your local timezone, but the GitHub API only shows UTC time so that is what is displayed by the Streak Stats.

The only way to correct this is by modifying your past commits or faking commit dates.

Also see https://github.com/DenverCoder1/github-readme-streak-stats/blob/main/docs/faq.md#why-doesnt-my-streak-stats-match-my-contribution-graph

To provide further explanation to the issue, this is how your contribution graphs appear when I view it. It shows no contributions on September 1, and the commit you showed appears to be on August 31.

![Screenshot_20220904-075158](https://user-images.githubusercontent.com/20955511/188317304-dfc6d3ce-0870-41d7-81c9-8fdb768746ee.png)
![Screenshot_20220904-075245](https://user-images.githubusercontent.com/20955511/188317307-2622236c-e1a2-419f-8335-b3c2e0a85c2c.png)

Note that the streak stats image goes through GitHub's servers and does not take timezone into account, so should display about the same for everyone.

So, in other words, both are accurate, but the GitHub API can't currently be relied on for localized info. You can leave it as is, or fix it by creating new commits dated September 1 (more details at the link above).

**Reply by Arpitchugh** [reply]
Thanks for helping out!!

**Reply by NathenaelTamirat** [reply]
but everytime for me when i try to open repository, edit comment, edit code... it doesn't count as a streak. but it displays on my profile as contribution. the streak only goes up when i push code. how can i solve that??

**Reply by Arpitchugh** [reply]
@NathenaelTamirat  That's how it works: only code that is pushed is counted toward the streak.

**Reply by NathenaelTamirat** [reply]
okay thank you very much
